### PR TITLE
first text of a general 'what is this ownCloud thing' item.

### PR DESCRIPTION
**Pull request by jospoortvliet**
please, @jancborchardt @DeepDiver1975 @karlitschek @dragotin @LukasReschke let me know what you think.

This was prompted by several people asking "but what IS this ownCloud thing then", AFTER they had been reading our website ;-)

Somehow a very dummy-proof description should be findable. I'll work on that findability later, just want to get the description right, first.

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

File: page-faq.php [hunk 1]
```diff
@@ -52,6 +53,17 @@
...
+<p>Many people prefer to have more control over who gets to see the photo's they send to their spouse or friends, who gets to read their bank account statements or determine their political allegiance. Or know where they are - as mobile phones track your location pretty much continuously...</p>
+<p>This is where ownCloud and similar technologies come in. ownCloud offers you the option to take back your data. Once you install it on a server (or let somebody do that for you!) you can access your private data in a easy to use web interface or synchronize it with your devices like Android and iPhones. You can store your contacts and calendar as well as files on ownCloud and, through the many ownCloud apps, use it to store passwords, play music and movies and so on. It also allows you to securely share with other people and collaborate on documents. Learn more <a href="/features">about ownCloud features here</a>.</p>
```
Comment (carlaschroder) [inline page-faq.php]: Please remove the apostrophe in 'photo's'. Plural, not possessive.

**Comment by carlaschroder** [comment]
A couple of suggestions. One, remove the reference to Edward Snowden because he is a polarizing figure, and a lot of good true blue American patriots (yes, that is sarcasm) think he is the devil. I suggest something like "the revelations that our own government is spying on us, and collecting and snooping into virtually all of our online communications." ("Criminals and large corporations" may be redundant, but let's not get derailed.)

Two, make the first sentence the answer to the question, something like "ownCloud is a file sharing server that puts the control and security of your own data back into your hands." The rest of the text already answers why should you care.

**Comment by carlaschroder** [comment]
This is great material. To a whole lot of people, computers and the Internet are black boxes, so anything we can do to teach them that it's not magic juice that they have no control over is excellent.

**Comment by jancborchardt** [comment]
Honestly I don't think this is clarified by adding _yet more_ text - but by improving the frontpage. The current "above the fold" view is beyond generic:

> Introducing
> ownCloud Server 8.0

and not very useful.

I'm not sure how much time I want to spend arguing about this though since the previous homepage made by @MariekevC and me (which I thought was superior and did a pretty good job explaining) was changed.

**Comment by jospoortvliet** [comment]
well, we have a product. Introducing that on our front page seems to make sense... I did change the 'learn more' link to go to features rather than the eight page, as I think that makes more sense. I also adjusted the features page to actually explain that ownCloud is a self hosted file sync and share server. With this PR, the text will be:

    ownCloud is a self hosted file sync and share server. It provides access
    to your data through a web interface or WebDAV while providing a platform
    to view, sync and share across devices easily—all under your control.
    ownCloud’s open architecture is extensible via a simple but powerful API
    for applications and plugins and works with any storage.

Comments welcome. And fwiw, I think the front page has improved

**Comment by jospoortvliet** [comment]
statistics may 25 2016
statistics may 25 2016
After the changes.

What do we think, a little better?

**Comment by jancborchardt** [comment]
The biggest thing on the page should not be the product name (especially not the long form "ownCloud Server 8.0" instead of simply "ownCloud 8").

The biggest thing should be the slogan. e.g. " Web services under your control" or whatever we choose as short slogan for the community. We could even just make the sentence "Access your data ..." thre focal point and remove the rest.

**Comment by jospoortvliet** [comment]
Jan suggested to have the slogan/explanation what ownCloud is be big, rather than the announcement of ownCloud 8. I think we should've done this a week or two after the release anyway...

statistics may 25 2016

What do we think?

**Comment by jospoortvliet** [comment]
(also made some further refinements to the features page. I'll give it more love and attention before ownCloud 8.1 comes out)

**Comment by carlaschroder** [comment]
> Jan suggested to have the slogan/explanation what ownCloud is be big, rather than the announcement of ownCloud 8

Sure, that makes sense. I like very much all the work that has gone into delivering useful information, rather than fluffy stuff :)

**Comment by karlitschek** [comment]
looks great :+1:

**Comment by jospoortvliet** [comment]
Going in and, after a check on staging.owncloud.org, going live!

**Comment by jancborchardt** [comment]
Yeah, looks a tad better now. We still have 5 equally large things fighting for attention though. We need to improve that. ; )

**Comment by jospoortvliet** [comment]
I'll work on the look of the 3 panes later.

**Pull request merged**